packages/core/src/GenericBanner/GenericBanner.spec.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { render, fireEvent } from 'testing-library'
+import { Warning, Emoticon } from 'pcln-icons'
+import { Text } from '../Text'
+import { Image } from '../Image'
+import { GenericBanner, theme } from '.'
+
+const props = {
+  p: 2,
+  heading: (
+    <Text.span fontWeight='bold' textColor='primary.base'>
+      COVID-19&nbsp;&nbsp;
+    </Text.span>
+  ),
+  text: (
+    <Text.span textColor='primary.base'>
+      Update: Your travel may be impacted. Please review this hotels important
+      info.&nbsp;&nbsp;
+    </Text.span>
+  ),
+  ctaText: (
+    <Text.span
+      fontSize={[0, null, null, 1]}
+      textColor='primary.base'
+      fontWeight='bold'
+    >
+      Read More
+    </Text.span>
+  ),
+  imageLeft: <Image alt='ImageLeft' />,
+  iconLeft: <Warning />,
+  iconRight: <Emoticon />,
+  color: 'caution.light',
+}
+
+describe('GenericBanner', () => {
```

Comment:
Can you add a test to assert that `fontSize` is being passed along everywhere that we expect it to be?